packages/docs/src/pages/en/getting-started/upgrade-guide.md

```diff
@@ -100,19 +119,28 @@ app.use(vuetify)
   - Allowed values are `'elevated'`, `'flat'`, `'tonal'`, `'outlined'`, `'text'`, or `'plain'`.
 - `text` prop has new purpose. It represents the text content of the alert, if default slot is not used.
 
-### v-btn
+### v-btn/v-btn-toggle
 
 - `active-class` prop has been renamed to `selected-class`.
-- `fab` is no longer supported.
+- `fab` is no longer supported. If you just need a round button, use `icon` prop or apply a `.rounded-circle` class
 - `flat` / `outlined` / `text` / `plain` props have been combined into a single `variant` prop.
 - `depressed` has been renamed to `variant="flat"`.
 - `retain-focus-on-click` has been removed, buttons use `:focus-visible` instead.
+- `v-btn-toggle` needs `mandatory="force"` prop to achieve the same behaviour as `mandatory` prop in v2
+- `v-btn` has a new size `x-large`, which is why you have to down-shift previously used sizes by one to get the same size as in v2. E.g. `small` becomes `x-small`
```

Comment:
xLarge existed in v2

Reply:
Can't reproduce exactly what the issues was. Removed.